task/build-vm-image/0.1/build-vm-image.yaml

```diff
@@ -0,0 +1,263 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/pipelines.minVersion: 0.12.1
+    tekton.dev/tags: image-build, appstudio, hacbs
```

Comment:
Do we still need `hacbs`?

Reply:
Almost certainly not. I can drop it and add konflux, or we can do a mass `sed` PR later to update it across the board?

Reply:
```
❯ git grep tekton.dev/tags | grep hacbs | wc -l
30
```
Let's do it in a separate PR and get them all at once. :)